src/Fields/field.jl

```diff
@@ -706,10 +699,10 @@ for reduction in (:sum, :maximum, :minimum, :all, :any, :prod)
                                    mask = get_neutral_mask(Base.$(reduction!)),
                                    dims = :)
 
+            conditioned_c = condition_operand(f, c, condition, mask)
             T = filltype(Base.$(reduction!), c)
             loc = reduced_location(location(c); dims)
             r = Field(loc, c.grid, T; indices=indices(c))
-            conditioned_c = condition_operand(f, c, condition, mask)
             initialize_reduced_field!(Base.$(reduction!), identity, r, conditioned_c)
             Base.$(reduction!)(identity, r, conditioned_c, init=false)
```

Comment:
@glwagner, shouldn't this be:

```suggestion
            initialize_reduced_field!(Base.$(reduction!), f, r, conditioned_c)
            Base.$(reduction!)(f, r, conditioned_c, init=false)
```

Reply:
Does it not need `f` there because `f` is incorporated in `conditioned_c`?